projects/element-ng/split/si-split-part.component.scss

```diff
@@ -7,13 +7,40 @@ $header-height: 40px;
   display: flex;
   flex-direction: column;
   overflow: hidden;
+  border-radius: all-variables.$card-border-radius;
+
+  // To have rounded corners but not be breaking and allow the user to change the background color
+  &.bg-base-1 {
+    background-color: transparent !important; // stylelint-disable-line declaration-no-important
+
+    .si-split-part-content {
+      background-color: all-variables.$element-base-1;
+    }
+  }
+
+  &.bg-base-0 {
+    background-color: transparent !important; // stylelint-disable-line declaration-no-important
+
+    .si-split-part-content {
+      background-color: all-variables.$element-ui-4;
```

Comment:
ui-4? that's for borders. did you mean base-0

Reply:
Nice catch, fixed